target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-ad7200.dts

```diff
@@ -0,0 +1,443 @@
+			status = "ok";
+		};
+
+		phy@110f8830 {		/* USB3 port 0 SS phy */
+			status = "ok";
+		};
+
+		usb30@0 {
+			status = "ok";
+		};
+
+		usb30@1 {
+			status = "ok";
+		};
+
+		pcie0: pci@1b500000 {
+			status = "ok";
+			phy-tx0-term-offset = <7>;
+		};
+
+		pcie1: pci@1b700000 {
+			status = "ok";
+			phy-tx0-term-offset = <7>;
+		};
+
+		pcie2: pci@1b900000 {
+			status = "ok";
+			phy-tx0-term-offset = <7>;
+		};
+
+		mdio0: mdio {
+			compatible = "virtual,mdio-gpio";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			gpios = <&qcom_pinmux 1 GPIO_ACTIVE_HIGH &qcom_pinmux 0 GPIO_ACTIVE_HIGH>;
+			pinctrl-0 = <&mdio0_pins>;
+			pinctrl-names = "default";
+
+			phy0: ethernet-phy@0 {
+				device_type = "ethernet-phy";
+				reg = <0>;
+				qca,ar8327-initvals = <
+					0x00004 0x7600000   /* PAD0_MODE */
+					0x00008 0x1000000   /* PAD5_MODE */
+					0x0000c 0x80        /* PAD6_MODE */
+					0x000e4 0x6a545     /* MAC_POWER_SEL */
+					0x000e0 0xc74164de  /* SGMII_CTRL */
+					0x0007c 0x4e        /* PORT0_STATUS */
+					0x00094 0x4e        /* PORT6_STATUS */
+					>;
+			};
+
+			phy4: ethernet-phy@4 {
+				device_type = "ethernet-phy";
+				reg = <4>;
+			};
+		};
+
+		gmac1: ethernet@37200000 {
+			status = "ok";
+			phy-mode = "rgmii";
+			qcom,id = <1>;
+
+			pinctrl-0 = <&rgmii2_pins>;
+			pinctrl-names = "default";
+
+			mtd-mac-address = <&defaultmac 0x8>;
+			mtd-mac-address-increment = <1>;
+
+			fixed-link {
+				speed = <1000>;
+				full-duplex;
+			};
+		};
+
+		gmac2: ethernet@37400000 {
+			status = "ok";
+			phy-mode = "sgmii";
+			qcom,id = <2>;
+
+			mtd-mac-address = <&defaultmac 0x8>;
+
+			fixed-link {
+				speed = <1000>;
+				full-duplex;
+			};
+		};
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		wifi {
+			label = "wifi";
+			gpios = <&qcom_pinmux 49 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&qcom_pinmux 64 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&qcom_pinmux 65 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+		ledgeneral {
+			label = "ledgeneral";
+			gpios = <&qcom_pinmux 16 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		lan {
+			label = "ad7200:white:lan";
+			gpios = <&qcom_pinmux 6 GPIO_ACTIVE_HIGH>;
+		};
+		usb4 {
+			label = "ad7200:white:usb_4";
+			gpios = <&qcom_pinmux 7 GPIO_ACTIVE_HIGH>;
+		};
+		usb2 {
+			label = "ad7200:white:usb_2";
+			gpios = <&qcom_pinmux 8 GPIO_ACTIVE_HIGH>;
+		};
+		wps {
+			label = "ad7200:white:wps";
+			gpios = <&qcom_pinmux 9 GPIO_ACTIVE_HIGH>;
+		};
+		wan_blue {
+			label = "ad7200:white:wan";
+			gpios = <&qcom_pinmux 33 GPIO_ACTIVE_LOW>;
+		};
+		power: status {
+			label = "ad7200:white:status";
+			gpios = <&qcom_pinmux 53 GPIO_ACTIVE_HIGH>;
+		};
+		ledgnr: ledgnr {
+			label = "ad7200:white:ledgnr";
+			gpios = <&qcom_pinmux 66 GPIO_ACTIVE_HIGH>;
+		};
+	};
```

Comment:
I suspect it turns off leds and it's a switch button. And button gpio a are also messed it seems 
`#define AP148_GPIO_BTN_JUMPSTART_AD7200		67
#define AP148_GPIO_BTN_RESET_AD7200		7
#define AP148_GPIO_BTN_WIRELESS_AD7200  	54
#define AP148_GPIO_BTN_LEDGENERAL_AD7200 	53

static struct gpio_keys_button ap148_gpio_keys_ad7200[] = {
	{
		.desc		= "wps",
		.type		= EV_KEY,
		.code		= KEY_WPS_BUTTON,
		.debounce_interval = AP148_KEYS_DEBOUNCE_INTERVAL,
		.gpio		= AP148_GPIO_BTN_JUMPSTART_AD7200,
		.wakeup		= 1,
		.active_low	= 1,
	},
	{
		.desc		= "wireless",
		.type		= EV_KEY,
		.code		= KEY_WLAN,
		.debounce_interval = AP148_KEYS_DEBOUNCE_INTERVAL,
		.gpio		= AP148_GPIO_BTN_WIRELESS_AD7200,
		.wakeup		= 1,
		.active_low	= 1,
	},
	{
		.desc		= "reset",
		.type		= EV_KEY,
		.code		= KEY_RESTART,
		.debounce_interval = AP148_KEYS_DEBOUNCE_INTERVAL,
		.gpio		= AP148_GPIO_BTN_RESET_AD7200,
		.wakeup		= 1,
		.active_low	= 1,
	},
	{
		.desc		= "ledgeneral",
		.type		= EV_KEY,
		.code		= KEY_DOLLAR,
		.debounce_interval = AP148_KEYS_DEBOUNCE_INTERVAL,
		.gpio		= AP148_GPIO_BTN_LEDGENERAL_AD7200,
		.wakeup		= 1,
		.active_low	= 1,
	}
};`